examples/sphere.py

```diff
@@ -169,33 +722,17 @@ def create_scheduler(algorithm,
     max_bound = solution_dim / 2 * 5.12
     bounds = [(-max_bound, max_bound), (-max_bound, max_bound)]
     initial_sol = np.zeros(solution_dim)
-    batch_size = 37
-    num_emitters = 15
     mode = "batch"
 
-    if algorithm in ["cma_mae", "cma_maega"]:
-        threshold_min = 0
-    else:
-        threshold_min = -np.inf
-
-    if algorithm in ["me_map_elites"]:
-        batch_size = 50
-        num_emitters = 48
+    config = CONFIG[algorithm]
 
     # Create archive.
-    if algorithm in ["cvt_map_elites", "line_cvt_map_elites"]:
-        archive = CVTArchive(solution_dim=solution_dim,
-                             cells=10_000,
-                             ranges=bounds,
-                             samples=100_000,
-                             use_kd_tree=True)
+    archive_module = importlib.import_module("ribs.archives")
```

Comment:
I appreciate the creativity with importlib here, but I think you can keep it simple by just using the relevant Python class e.g. `{"class": GridArchive}`